src/app/modules/shared_modules/keycard_popup/internal/copying_keycard_state.nim

```diff
@@ -0,0 +1,53 @@
+type
+  CopyingKeycardState* = ref object of State
+
+proc newCopyingKeycardState*(flowType: FlowType, backState: State): CopyingKeycardState =
+  result = CopyingKeycardState()
+  result.setup(flowType, StateType.CopyingKeycard, backState)
+
+proc delete*(self: CopyingKeycardState) =
+  self.State.delete
+
+proc buildKeypairAndAddToMigratedKeypairs(self: CopyingKeycardState, controller: Controller) =
+  let cardMetadata = controller.getMetadataForKeycardCopy()
+  var addresses: seq[string]
+  for wa in cardMetadata.walletAccounts:
```

Comment:
It's can be a bit easier using sequtils: 
`let addresses = cardMetadata.walletAccounts.map(wa => wa.address)`